modules/mobile/app/models/mobile/v0/adapters/vaos_v2_appointments.rb

```diff
@@ -115,6 +126,17 @@ def build_appointment_model(appointment_hash)
         end
         # rubocop:enable Metrics/MethodLength
 
+        def patient_phone_number(appointment_hash)
+          phone_number = contact(appointment_hash.dig(:contact, :telecom), CONTACT_TYPE[:phone])
+
+          return nil unless phone_number
+
+          parsed_phone = parse_phone(phone_number)
+          joined_phone = "#{parsed_phone[:area_code]}-#{parsed_phone[:number]}"
```

Comment:
This assumes that if there is a non-nil phone number there will always be an area code and number. This seems like a safe enough assumption. Test data all has both numbers.